GVRf/Framework/framework/src/main/java/org/gearvrf/GVRActivity.java

```diff
@@ -66,11 +66,11 @@
     private volatile GVRConfigurationManager mConfigurationManager;
     private GVRMain mGVRMain;
     private VrAppSettings mAppSettings;
-    private static View mFullScreenView = null;
+    private static View mFullScreenView;
```

Comment:
Well, this is old code where I just removed the redundant initialization. Probably doesn't have to be static but have not looked at it.